test/support/factory.ex

```diff
@@ -883,4 +885,18 @@ defmodule Trento.Factory do
       to_package_id: "#{RandomElixir.random_between(0, 1000)}"
     }
   end
+
+  def user_factory do
+    password = Faker.Pokemon.name()
+
+    %User{
+      email: Faker.Internet.email(),
```

Comment:
I hope that the uniqueness of username and email doesn't raise any issue here. Hopefully the faker dictionaries have enough emails hehe
Otherwise we can add `sequence` in the future